templates/job-dashboard.php

```diff
@@ -87,7 +87,14 @@
 				<?php endforeach; ?>
 			<?php endif; ?>
 		</tbody>
-		<?php if ( $submit_job_form_page_id && ( job_manager_count_user_job_listings() < $submission_limit || ! $submission_limit ) ) : ?>
+		<?php
+
+var_dump($submit_job_form_page_id );
+var_dump(  job_manager_count_user_job_listings() < $submission_limit );
+var_dump( $submission_limit );
+
+		?>
+		<?php if ( $submit_job_form_page_id && job_manager_count_user_job_listings() < $submission_limit && false !== $submission_limit ) : ?>
```

Comment:
I think the logic is not right? It should be
- if there is a `$submit_job_form_page_id`
- AND ( the job count is smaller than the limit, OR there is no limit )

Not sure if you saw my other comment:

> Looks like this option is also used here where the same change should happen.
https://github.com/Automattic/WP-Job-Manager/blob/7411e91b491475e841a65c3b13b31132727c69f3/includes/class-wp-job-manager-shortcodes.php#L110-L114
> 
> And is there a similar check on the actual job submission page and endpoint?